lib/pollynomial/synthesizer.rb

```diff
@@ -38,9 +38,8 @@ def synthesize(text, file_name: "tmp.mp3")
       end
     end
 
-    def available_voices_in(language_code: 'en-US')
-      voices = client.describe_voices(language_code: language_code)
-      voices.voices if voices
+    def voices(language=nil)
+      client.describe_voices(language_code: language).voices
```

Comment:
In which use case do you think it returns nil? If you send an unrecognized language to AWS, an exception is raised either way. Any use case I missed?